version.php

```diff
@@ -23,7 +23,7 @@
 
 defined('MOODLE_INTERNAL') || die();
 
-$plugin->version   = 2016011300;
+$plugin->version   = 2016042500;
```

Comment:
Did you get both the upgrade and the task list to get registered successfully by bumping the version?